layouts/partials/head_includes.html

```diff
@@ -46,7 +46,7 @@
 {{ if .Site.Params.faviconfile }}
     href="/{{ .Site.Params.faviconfile }}"
 {{ else }}
-    href="/img/favicon.ico"
+    href="{{ .Site.BaseURL  }}img/favicon.ico"
```

Comment:
Not sure if you noticed but there is an extra white space after `BaseURL`. 😅
Thanks for your PR, it helped me to solve a problem.